requirements.txt

```diff
@@ -28,3 +28,4 @@ pycocotools>=2.0.2
 py-cpuinfo>=8.0.0
 shapely>=2.0.0
 fire==0.5.0
+Pillow<=9.5.0
```

Comment:
In the ultralytics requirements, it seems more appropriate to use '==' instead of '<=' since it specifies 'pillow>=7.1.2'
refer: https://github.com/ultralytics/ultralytics/blob/main/requirements.txt